mart/attack/perturber/perturber.py

```diff
@@ -46,10 +48,11 @@ def __init__(
         self.projector = projector
         self.optim_params = optim_params
 
-        self.perturbation = torch.nn.UninitializedParameter()
+        # Pre-occupy the name of the buffer.
+        self.register_buffer("perturbation", torch.nn.UninitializedBuffer(), persistent=False)
```

Comment:
Why is it necessary to "pre-occupy" the name of the buffer?

Reply:
So that `extra_repr()` always get perturbation.

Reply:
Can you add that to the comment?

Reply:
Commented.